data/json/furniture_and_terrain/furniture-plumbing.json

```
@@ -80,7 +80,7 @@
     "name": "toilet",
     "symbol": "&",
     "color": "white",
-    "description": "A porcelain throne.  Emergency water source, from the tank, and provider of relief.",
+    "description": "An invaluable appliance in any environment, it would be a miracle to have one that works.  The standing tank holds a moderate amount of water, but while better than anything that would be in the bowl, it's not the cleanest.",
```

```suggestion
    "description": "An invaluable appliance in any environment, it would be a miracle to have one that works.  The standing tank may hold a moderate amount of water, but while better than anything that would be in the bowl, it would not be the cleanest.",
```
The storage tank does not always contain water.

---

Ah yeah, I forgot that's not how all toilets work.